### PR TITLE
add loading screen. Fixes issue 9

### PR DESCRIPTION
**Pull request by belenwebdev**
I added a loading screen to fix issue #9

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -3,13 +3,16 @@ import Image from 'next/image'
...
+import React from 'react'
+import Loading from './components/Loading'
 
 export default function Home() {
   const router = useRouter()
   const { matches } = useMediaQueries({
     md: 'only screen and (min-width: 768px)',
     xxl: 'only screen and (min-width: 1536px)',
   })
+  const [isLoading, setLoading] = React.useState(true)
```
Comment (kirandash) [inline app/page.tsx:15]: Please change the default loading state to `false`. 

It is not changing to false anywhere and hence showing the loader on home page forever.

```diff
- const [isLoading, setLoading] = React.useState(true)
+ const [isLoading, setLoading] = React.useState(false)
```

**Review by kirandash — CHANGES_REQUESTED** [review]

File: app/page.tsx (hunk 1 above)
Comment (belenwebdev) [reply app/page.tsx]: Oops! Sorry! I set it to true for easier testing of the styles, and I forgot to change it back.

**Review by kirandash — APPROVED** [review]
Thanks a lot 🙏🏻

File: app/page.tsx (hunk 1 above)
Comment (kirandash) [reply app/page.tsx]: yes, that's one of "THE" common mistake

happens with everyone 😅

**Pull request merged**